src/libraries/Microsoft.NETCore.Platforms/src/runtime.json

```diff
@@ -1728,6 +1728,17 @@
         "linux-x64"
       ]
     },
+    "haiku": {
```

Comment:
Update [PortableRuntimeIdentifierGraph.json](https://github.com/dotnet/runtime/commit/3b9b4fd5fcd3ff9ed0ea1622ff42f98a1068316b#diff-588ddddd3493c7445abeb05129392c25b9d03a30133085f6ec4b78d2b1adbe49) as well, which was recently moved in the repo?

Reply:
> Update [PortableRuntimeIdentifierGraph.json](https://github.com/dotnet/runtime/commit/3b9b4fd5fcd3ff9ed0ea1622ff42f98a1068316b#diff-588ddddd3493c7445abeb05129392c25b9d03a30133085f6ec4b78d2b1adbe49) as well, which was recently moved in the repo?

Probably not, because before it moved here, https://github.com/dotnet/sdk/pull/35231 containing the changes for Haiku in the portable graph has already been merged.

Reply:
See also: https://github.com/dotnet/runtime/blob/bbe5c66c50201816c14149fd3f259db86bc1f493/src/libraries/Microsoft.NETCore.Platforms/src/PortableRuntimeIdentifierGraph.json#L67-L77